### PR TITLE
ADEN-1978 INCONTENT_* ads refactoring

### PR DESCRIPTION
**Pull request by gabrys**

### CODE REVIEW

File: extensions/wikia/AdEngine/js/slot/inContent.js [hunk 1]
```diff
@@ -0,0 +1,319 @@
...
+			return false;
+		}
+
+		firstElementClear = section.$firstElement.css('clear');
+		if (firstElementClear === 'both' || firstElementClear === 'right') {
+			logInfo(section, 'NO: the first element in section has an incompatible clear CSS property');
+			return false;
+		}
+
+		maxHeightDiff = 1.2 * $ad.outerWidth(true) * adHeight / originalContainerWidth;
```
Comment (nandy-andy) [inline extensions/wikia/AdEngine/js/slot/inContent.js]: Where the `1.2` came from?

File: extensions/wikia/AdEngine/js/slot/inContentDesktop.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+				log('Capable section found, injecting the ad ' + slotName +
+					' between ' + section.name + ' and ' + section.nextName, 'info', logGroup);
+
+				$ad.find('.wikia-ad').attr('id', slotName);
+				inContent.injectAdPlaceholderIntoSection(section, $ad);
+				win.adslots2.push(slotName);
+
+				trackedData.tried300x600 = try300x600 ? 'yes' : 'no';
+				trackedData.fitted300x600 = slotName === 'INCONTENT_1C' ? 'yes' : 'no';
+				trackedData.top = $ad.offset().top.toFixed(-2);
```
Comment (nandy-andy) [inline extensions/wikia/AdEngine/js/slot/inContentDesktop.js]: Does it work with `-2`? AFAIK `toFixed()` accepts as argument number between `0` and `20`. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed

File: extensions/wikia/AdEngine/js/slot/inContent.js [hunk 3]
```diff
@@ -0,0 +1,319 @@
...
+				section.keepAd = true;
+			}
+		}
+	}
+
+	/**
+	 * Clean up all the extra CSS classes added in case the ad was not safe to leave.
+	 *
+	 * DOM/layout querying: forbidden
+	 * DOM modification:    OK
```
Comment (nandy-andy) [inline extensions/wikia/AdEngine/js/slot/inContent.js]: I like the guidelines :+1:

File: extensions/wikia/AssetsManager/config.php [hunk 4]
```diff
@@ -118,7 +118,7 @@
 
 $config['adengine2_taboola_js'] = array(
 	'type' => AssetsManager::TYPE_JS,
-	'skin' => [ 'oasis', 'venus', 'wikiamobile' ],
```
Comment (nandy-andy) [inline extensions/wikia/AssetsManager/config.php]: Clean-up, clean-up :bathtub: :bath: :shower:

**Comment by fraszczakszymon** [comment]
LGTM :+1:

**Comment by nandy-andy** [comment]
LGTM :+1:

**Pull request merged**